spec/components/parameters/reports/flow/sessionId.ts

```diff
@@ -0,0 +1,16 @@
+import { ParameterObject } from 'openapi3-ts';
+import { createComponentRef } from '../../../../../utils/ref';
+
+const sessionId: ParameterObject = {
+  name: 'sessionId',
+  in: 'query',
+  required: false,
+  description: 'This is the ID between the flow and the client.',
```

Comment:
"client" não é a nomenclatura que usamos, e se fosse, o correto seria "customer", mas eu usaria user. Antes eu diria pra chamar de conversa, mas agora que não tá mais preso a conversa, acho eu deria uma explicada maior. Tipo:
"This ID represents a flow execution. In case of conversational flows, it identifies the conversation in our platform."

Reply:
O ainda:
"This ID represents a flow execution (session). In case of conversational flows, it identifies the conversation in our platform."

Reply:
Sugeri por o (session) pra fazer sentido o "The session identifier." que está na descrição da resposta. Talvez fique melhor esse texto aqui também estar lá.